test/e2e/config/provider.yaml

```diff
@@ -75,6 +75,8 @@ providers:
         files:
           # Add a cluster template
           - sourcePath: "../data/infrastructure-provider-byoh/v1beta1/templates/e2e/cluster-template.yaml"
+          - sourcePath: "../data/infrastructure-provider-byoh/v1beta1/templates/e2e/clusterclass-quickstart.yaml"
```

Comment:
Not sure if this is the right way to provide a path, Is there a way we can provide `basepath` in this template?

Reply:
For uniformity keeping it same as other providers, referred [this](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/data/e2e_conf.yaml#L209) .